src/main/java/com/google/sps/servlets/RSAServlet.java

```diff
@@ -0,0 +1,35 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/rsa-data")
+public class RSAServlet extends HttpServlet {
```

Comment:
The code looks pretty good. One thing you can improve on is to add a JavaDoc to the class. Per the Google style guide, all public classes / methods / fields must have JavaDoc. I don't think you need to add ones to doGet/doPost since they're standard and should be already described in the superclass.